raphtory-graphql/src/model/mod.rs

```diff
@@ -136,16 +149,20 @@ impl QueryRoot {
             Err(InvalidPathReason::NamespaceDoesNotExist(path))
         }
     }
-
+    /// Returns root namespace
+    ///
+    /// Returns::  Root namespace
     async fn root<'a>(ctx: &Context<'a>) -> Namespace {
         let data = ctx.data_unchecked::<Data>();
         Namespace::new(data.work_dir.clone(), data.work_dir.clone())
     }
-
+    /// Returns a plugin.
```

Comment:
I think you want to clarify here that this returns the the collection of query plugins, not an individual plugin